src/main/java/gov/usds/case_issues/db/model/OAuthUser.java

```diff
@@ -0,0 +1,49 @@
+package gov.usds.case_issues.db.model;
+
+import java.time.ZonedDateTime;
+
+import javax.persistence.Entity;
+import javax.validation.constraints.NotNull;
+
+import org.hibernate.annotations.NaturalId;
+
+/**
+ * Metadata about the case issue data stored in this system
+ */
+@Entity
+public class OAuthUser extends UpdatableEntity {
+
+	@NotNull
+	private ZonedDateTime lastSeen;
+	@NotNull
+	private String printName;
+	@NaturalId
+	@NotNull
+	private String userId;
+
+	protected OAuthUser() {
```

Comment:
Is there an advantage to making this private? Right now `CaseIssueUpload` is defined as `private` and all other models are defined as `protected`.

Would be nice to settle on a single model boilerplate

Reply:
The only advantage to making it `private` is to prevent people from using it in contexts where it shouldn't be used—`protected` does the job pretty well and doesn't result in weird shenanigans with `@SuppressWarnings`, but may cause it to show up as an available constructor in your IDE sometimes when you probably didn't want it to. Since there is no context in which we want a subclass or another class in this package calling the no-arg constructor, `private` is more "correct", which is why I said "probably should", but it's not a big deal either way.